scripts/bump-version.sh

```diff
@@ -9,5 +9,9 @@ OLD_VERSION="$1"
 NEW_VERSION="$2"
 
 sed -i -e 's/^version = .*$/version = '"$NEW_VERSION/" setup.cfg
+# only bump this on tagged releases
+if [[ "$NEW_VERSION" != *.dev0 ]]; then
+  sed -i -e 's/^SELF_HOSTED_STABLE_VERSION = .*/SELF_HOSTED_STABLE_VERSION = "'"$NEW_VERSION"'"/' src/sentry/conf/server.py
```

Comment:
Do we use the word "stable" for this elsewhere? It's not really a stable version, just a somewhat arbitrary point in time.

Reply:
Iirc we use "latest" for this in e.g. Docker Hub.

Reply:
(Which is perennially confusing to me because I would expect "latest" to mean HEAD but that's a lot of inertia to overcome (we tweaked this in Codecov iirc).)

Reply:
So I guess I'm suggesting ...

```suggestion
  sed -i -e 's/^SELF_HOSTED_LATEST_VERSION = .*/SELF_HOSTED_LATEST_VERSION = "'"$NEW_VERSION"'"/' src/sentry/conf/server.py
```

Reply:
the beacon json blob is `{..., "version": {"stable": ...}}`

Reply:
Well there it is. This isn't user-visible, correct?

Reply:
Good enough.